episode-6.msrt

```diff
@@ -353,10 +351,8 @@
 
 71
 00:04:40,780 --> 00:04:47,710
-[ru] [ВАРИАНТ1:] Что тут опасного? Да ты просто боишься, что твое зелье  не сможет сравниться с моим!
-[ru] [ВАРИАНТ2:] Какая разница, что там твое зелье делает? Всем уже и так понятно, что я выиграла конкурс!
-[en] [1-st version] What could be so dangerous? You're just scared because your potion isn't as fantastic as mine!
-[en] [2-nd version] No matter what your potion does, everyone already knows that I've won the contest!
+[ru] Какая разница, что там твое зелье делает? Всем уже и так понятно, что я выиграла конкурс!
+[en] No matter what your potion does, everyone already knows that I've won the contest!
```

Comment:
Please keep both versions here. It is not yet clear which version will be accepted to final mix. We will ask actors to record both versions and then choose which would fit better for general animation flow.